File: Leetcode/JS/Leetcode1629-按键持续时间最长的键.js
var slowestKey = function(releaseTimes, keysPressed) {
    let ans = keysPressed[0],time = releaseTimes[0];
    for(let i = 0;i < releaseTimes.length;i++){
        const t = releaseTimes[i] - releaseTimes[i - 1];
        if(t > time || (t == time && ans < keysPressed[i])) time = t,ans = keysPressed[i]
    }
    return ans;
};

File: Leetcode/PY/Leetcode1629-按键持续时间最长的键.py
class Solution:
    def slowestKey(self, releaseTimes: List[int], keysPressed: str) -> str:
        keysPressed = list(keysPressed)
        ans,time = keysPressed[0],releaseTimes[0]
        for i in range(1,len(releaseTimes)):
            t = releaseTimes[i] - releaseTimes[i - 1]
            if t > time or (t == time and ans < keysPressed[i]): ans,time = keysPressed[i],t
        return ans